features/distroless_images.md

```diff
@@ -183,29 +167,31 @@ N/A
 
 - [x] Tests exist with the feature enabled that can be integrated with our automated performance testing.
 
+YES - The full Istio integration test suite is ran on each PR.
+
 **API**
 
-- [ ] TOC has reviewed the API and determined it to be complete. 
+- [x] TOC has reviewed the API and determined it to be complete. 
 
-Like NA, as there is no API, unless we consider the docker tag name an API. Will leave to TOC to decide.
+YES - only a minor API in ProxyConfig which was approved.
 
 **Tooling**
 
-- [ ] Any necessary tooling to use/debug the feature has been implemented and is complete. 
+- [x] Any necessary tooling to use/debug the feature has been implemented and is complete. 
 
-NO - we essentially depend on Kubernetes Ephemeral Containers to be in widespread use in order to facilitate debugging of the proxy or control plane once distroless images are in use. This is apparently beta in Kubernetes 1.21.
+YES - we essentially depend on Kubernetes Ephemeral Containers and document how to sue it
 
 **Bugs**
 
 - [ ] Feature has no known major issues.
 
-No: https://github.com/istio/istio/issues?q=is%3Aissue+distroless+is%3Aopen+sort%3Aupdated-desc
+No: https://github.com/istio/istio/issues/37235. bug-report does not fully support this mode.
 
 **Approvals**
 
-- [ ] The appropriate work group(s) have reviewed and approved promotion of the feature.
+- [x] The appropriate work group(s) have reviewed and approved promotion of the feature.
 - [ ] The supportability review panel has reviewed promotion of the feature.  
-- [ ] The TOC has reviewed and approved promotion of the feature as part of the
+- [x] The TOC has reviewed and approved promotion of the feature as part of the
```

Comment:
If this PR is approved, this will be true. It has not been approved outside of this PR.